src/ExaTioga.cpp

```diff
@@ -34,6 +34,8 @@ ExaTioga::ExaTioga(stk::ParallelMachine& comm)
     , m_stk(comm)
     , m_amr()
     , m_tioga(tioga_nalu::TiogaRef::self().get())
+    , m_doWrite(true)
+    , m_outputFreq(1)
```

Comment:
```suggestion
```

Reply:
@sayerhs This has be rectified in 
eac195f